assembly/assembly-ide-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -70,6 +70,7 @@ docker.registry.auth.password=NULL
 docker.registry.auth.email=NULL
 docker.connection.tcp.connection_timeout_ms=600000
 docker.connection.tcp.read_timeout_ms=600000
+docker.clean.up.unused.containers.period_min=5
```

Comment:
It is good that you've added unit to property name. But naming is not the best. We use dots to separate components and underscores to separate words of one component.